<ui version="4.0" >
 <class>mNote</class>
 <widget class="QDialog" name="mNote" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Notizblatt</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>  Notizblatt</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_2" >
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>( Notiz - Zeilenlänge max. 255 Buchstaben )</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QTableWidget" name="NoteTable" >
     <column>
      <property name="text" >
       <string>Index             </string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Notiz                                                                                                           </string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="ButtonDelete" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>Löschen</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>19</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="ButtonEnde" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>175</red>
           <green>215</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>175</red>
           <green>215</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>175</red>
           <green>215</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>                        F10   -  Ende                   </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
